cypress-tests/cypress/support/commands.js

```diff
@@ -722,6 +655,14 @@ Cypress.Commands.add("handleRedirection", (globalState, expected_redirection) =>
             cy.get('#test-source-authorize-3ds').click();
           })
       })
+  } else if (connectorId == "adyen") {
+    cy.get('iframe')
+      .its('0.contentDocument.body')
+      .within((body) => {
+        cy.get('input[type="password"]').click();
+        cy.get('input[type="password"]').type("password");
+        cy.get('#buttonSubmit').click();
+      })
```

Comment:
is handling redirection for Adyen is also included? if yes please add the test screenshots

Reply:
I copy pasted from #4396 since it was working. @Gnanasundari24 picked the changes from that I guess in https://github.com/juspay/hyperswitch/pull/4454. 

I'll revert this change.